tools/llvm-slicer.cpp

```diff
@@ -99,6 +100,10 @@ enum PtaType {
     fs, fi
 };
 
+enum class RdaType {
```

Comment:
Here we need to use just enum, not enum class, otherwise we must pass the whole classified name to the argument, i.e. `-rda RdaType::ss` instead of `-rda ss`